draft-ietf-httpbis-optimistic-upgrade.md

```diff
@@ -157,7 +158,17 @@ In HTTP/1.1, proxy clients that send CONNECT requests on behalf of untrusted TCP
 
 Proxy clients that don't implement at least one of these two behaviors are vulnerable to a trivial request smuggling attack ({{request-smuggling}}).
 
-At the time of writing, some proxy clients are believed to be vulnerable as described.  When communicating with potentially vulnerable clients, proxy servers MUST close the underlying connection when rejecting an HTTP/1.1 CONNECT request, without processing any further data on that connection, whether or not the request headers include "Connection: close".  Note that this mitigation will frequently impair the performance of correctly implemented clients, especially when returning a "407 (Proxy Authentication Required)" response.  This performance loss can be avoided by using HTTP/2 or HTTP/3, which are not vulnerable to this attack.
+At the time of writing, some proxy clients are believed to be vulnerable as described.  As a mitigation, this document updates RFC 9112 by adding the following section:
+
+> 9.6.1.  Rejecting a CONNECT Request
```

Comment:
Are the requirements on clients already stated in RFC 9112? If not, I'd make those an update to 9112 as well. I don't know that it's required to inject an explicit section, though that's fine. I think you can also simply say "{{RFC9112}} is updated to state that..." if you prefer.

Reply:
OK, I restructured this to make this text all part of the update.